ios/FluentUI/Table View/TableViewHeaderFooterView.swift

```diff
@@ -365,15 +365,6 @@ open class TableViewHeaderFooterView: UITableViewHeaderFooterView {
     ///   - leadingView: An optional custom view that appears near the leading edge of the view.
     private func setup(style: Style, accessoryButtonTitle: String, leadingView: UIView? = nil) {
         updateTitleViewFont()
-        switch style {
-        case .header, .divider, .dividerHighlighted, .headerPrimary:
```

Comment:
this is quite a bit of behavior regression that we can't use voiceover rotor to navigate around header traits.

Reply:
+1 - can we keep the header traits but get rid of the isAccessibilityElement? Worst case that is a clear improvement and makes the VoiceOver bug much more rare. Best case the new bug is not a real scenario in any clients consuming this control